CNAME

```diff
@@ -1 +0,0 @@
-debezium.io
```

Comment:
Is this going elsewhere? It's definitely needed in the output published to GH Pages.